### PR TITLE
Add set_capture_pts support to the Vulkan backend

### PR DESCRIPTION
**Pull request by mbouron-gpsw**

### CODE REVIEW

File: libnodegl/backends/vk/vkcontext.c [hunk 1]
```diff
@@ -786,6 +787,15 @@ struct vk_extension {
             {0},
         },
     },
+    {
```
Comment (cboesch-gpsw) [inline libnodegl/backends/vk/vkcontext.c:790]: style: can you merge this line with the previous one?

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ Versioning](https://semver.org/spec/v2.0.0.html) for `libnodegl`.
 - `ngl-diff` tool to visually compare videos
 - `UniformColor.space` colorspace configuration (sRGB, HSL, HSV)
 - `AnimatedColor` and `AnimKeyFrameColor` nodes
+- `set_surface_pts` support to the Vulkan backend
```
Comment (cboesch-gpsw) [inline CHANGELOG.md:17]: Do you mind adding "(Android)"; it can be confusing for the casual user otherwise

**Review by cboesch-gpsw — APPROVED** [review]
2 small comment, LGTM otherwise

File: CHANGELOG.md (hunk 2 above)
Comment (mbouron-gpsw) [reply CHANGELOG.md]: Done (also added to the commit message).

File: libnodegl/backends/vk/vkcontext.c (hunk 1 above)
Comment (mbouron-gpsw) [reply libnodegl/backends/vk/vkcontext.c]: Done.

**Pull request merged**